### PR TITLE
update ch.bfe.energieforschung

### PR DESCRIPTION
**Pull request by daguer**

### CODE REVIEW

File: chsdi/models/vector/uvek.py [hunk 1]
```diff
@@ -488,64 +488,42 @@ class ENERGIESTAEDTE2000AUFDEMWEG(Base, Vector):
 
 
 class ENERGIEFORSCHUNG(Base, Vector):
-    __tablename__ = 'energieforschung'
+    __tablename__ = 'energieforschung_new'
```
Comment (gjn) [inline chsdi/models/vector/uvek.py]: Is this a new layer? Or replacing existing layer?

Does that mean that old links with the old technical name won't work anymore?

**Comment by gjn** [comment]
In future, can someone from dataintegration do a review as well?

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -2217,64 +2217,13 @@ msgid "ch.bafu.laerm-strassenlaerm_tag"
...
 
 msgid "ch.bafu.landesforstinventar-baumarten"
-msgstr "Baumarten (LFI)"
+msgstr "Spezias da bostgs (IFN)"
 
 msgid "ch.bafu.landesforstinventar-totholz"
-msgstr "Totholz (LFI)"
+msgstr "Laina morta (IFN)"
 
 msgid "ch.bafu.landesforstinventar-waldanteil"
```
Comment (gjn) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: Is it correct to remove all those translations below? And why only for fi and not the other languages?

**Comment by gjn** [comment]
When doing translations, look at the diff..

File: chsdi/models/vector/uvek.py (hunk 1 above)
Comment (gjn) [inline chsdi/models/vector/uvek.py]: Stupid me, technical layer name didn't, so ignore my comment.

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po (hunk 2 above)
Comment (daguer) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: I have only change one layer (energieforschung).
May be a problem with the rebase of my branch...
I do it again on Monday...

Comment (loicgasser) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: Translations have been doubled. Translations are there so it is correcting a bad rebase. LGTM to me.

**Comment by loicgasser** [comment]
jenkins test

**Comment by loicgasser** [comment]
Jenkins error is unrelated see: https://github.com/geoadmin/mf-chsdi3/pull/1568

**Pull request merged**